ThunderRequest/TSCRequest.m

```diff
@@ -9,8 +9,15 @@
 #import <AppKit/AppKit.h>
 #endif
 
+@import os.log;
+static os_log_t request_log;
+
 @implementation TSCRequest
 
++ (void)initialize {
```

Comment:
Unsure if having this in the initialize method is necessary. Is there a better place to have it?

Reply:
See apple example, this is the technique they use: `https://developer.apple.com/library/content/samplecode/Logging/Listings/Paper_Company_Paper_Company_APLViewController_m.html#//apple_ref/doc/uid/TP40017510-Paper_Company_Paper_Company_APLViewController_m-DontLinkElementID_9`